test/sharness/t0250-files-api.sh

```diff
@@ -358,6 +358,25 @@ test_files_api() {
 		verify_dir_contents /
 	'
 
+	# test truncating
+	test_expect_success "create a new file" '
+		echo "some content" | ipfs files write --create /cats
```

Comment:
Perhaps also verify its contents are as expected before doing the truncate below?